src/components/SwipeableItem.tsx

```diff
@@ -1,71 +1,64 @@
-import React from 'react';
-import { Icon, Text } from '@ui-kitten/components';
 //https://medium.com/async-la/swipe-to-delete-with-reanimated-react-native-gesture-handler-bd7d66085aee
+import React, { ReactNode } from 'react';
 import { PanGestureHandler, State as GestureState } from 'react-native-gesture-handler';
 import Animated from 'react-native-reanimated';
 
 const {
-    multiply,
     min,
-    add,
-    divide,
     event,
     cond,
     Value,
     block,
     set,
     eq,
-    not,
     clockRunning,
     and,
     startClock,
     stopClock,
     spring,
     greaterThan,
-    lessThan,
+    interpolate,
     call,
     Clock,
 } = Animated;
 
-interface SwipeRowProps {
+interface SwipeableItemProps {
     swipeThreshold: number;
     onSwipe: (taskId: number) => void;
     itemId: number;
     children?;
+    leftIcon?: (string) => ReactNode;
 }
 
-class SwipeRow extends React.Component<SwipeRowProps> {
+class SwipeableItem extends React.Component<SwipeableItemProps> {
     state = {
         iconColor: '#000000',
     };
 
```

Comment:
This file had a bunch of changes to remove the animation lag. Naming could probably be better idk